boards/fruitjam/magic.json

```diff
@@ -0,0 +1,51 @@
+{
+  "exportVersion": "1.0.0",
+  "exportedBy": "Adafruit",
+  "exportedAt": "2025-08-13T17:48:53.128Z",
+  "exportedFromDevice": {
+    "board": "fruitjam",
+    "firmwareVersion": "1.0.0-beta.111"
+  },
+  "components": [
+    {
+      "name": "NeoPixel",
+      "pixelNumber": 5,
+      "pixelsOrder": "GRB",
+      "pixelBrightness": 255,
+      "pinName": "D32",
+      "type": "neopixel",
+      "pixelsType": "NEOPIXEL",
+      "isPixel": true
+    },
+    {
+      "name": "Push Button",
+      "pinName": "D0",
+      "type": "push_button",
+      "mode": "DIGITAL",
+      "direction": "INPUT",
+      "period": 0,
```

Comment:
Don't think we add period for buttons, in all 3 of them, probably the exporter needs tweaking.